### ISSUE TITLE
FYI: a purple theme

### ISSUE THREAD

**Issue by jue89** [dune3d/dune3d#23]
... if you like it, you may add it to the color themes ;-)

![Screenshot from 2023-11-13 23-28-58](https://github.com/dune3d/dune3d/assets/6105784/967da0eb-587b-4bf2-85ec-b8c879df4e1e)

![Screenshot from 2023-11-13 23-29-36](https://github.com/dune3d/dune3d/assets/6105784/fb7978b2-2de1-4123-be3e-40c5b4c71c77)

<details>
<summary>prefs.json</summary>

```json
    "canvas": {
        "colors": {
            "BACKGROUND_BOTTOM": {
                "b": 0.08915555477142334,
                "g": 0.1133333370089531,
                "r": 0.1133333370089531
            },
            "BACKGROUND_TOP": {
                "b": 0.3566666543483734,
                "g": 0.0939222127199173,
                "r": 0.20427489280700684
            },
            "CONSTRAINT": {
                "b": 0.6800000071525574,
                "g": 0.25,
                "r": 0.6899999976158142
            },
            "CONSTRUCTION_ENTITY": {
                "b": 0.36000001430511475,
                "g": 0.20999999344348907,
                "r": 0.36000001430511475
            },
            "CONSTRUCTION_POINT": {
                "b": 0.7066666483879089,
                "g": 0.3038666546344757,
                "r": 0.31997865438461304
            },
            "ENTITY": {
                "b": 1.0,
                "g": 0.7366666793823242,
                "r": 0.9315333962440491
            },
            "HIGHLIGHT": {
                "b": 1.0,
                "g": 0.6166666746139526,
                "r": 0.7700000405311584
            },
            "HOVER": {
                "b": 1.0,
                "g": 0.8957332968711853,
                "r": 0.7733333110809326
            },
            "INACTIVE_ENTITY": {
                "b": 0.3400000035762787,
                "g": 0.24480000138282776,
                "r": 0.3158447742462158
            },
            "INACTIVE_POINT": {
                "b": 0.42666667699813843,
                "g": 0.024177780374884605,
                "r": 0.2898204028606415
            },
            "OTHER_BODY_SOLID_MODEL": {
                "b": 1.0,
                "g": 0.6966666579246521,
                "r": 0.7724999785423279
            },
            "POINT": {
                "b": 1.0,
                "g": 0.0,
                "r": 0.619999885559082
            },
            "SELECTED": {
                "b": 0.7399997711181641,
                "g": 0.0,
                "r": 1.0
            },
            "SELECTED_HOVER": {
                "b": 0.559999942779541,
                "g": 0.0,
                "r": 1.0
            },
            "SELECTION_BOX": {
                "b": 1.0,
                "g": 1.0,
                "r": 1.0
            },
            "SOLID_MODEL": {
                "b": 1.0,
                "g": 0.8117647171020508,
                "r": 0.8569411635398865
            }
        },
        "line_width": 1.0,
        "msaa": 4
    }
```

</details>

**Comment by carrotIndustries** [comment]
Look good, thanks! I'll eventually add this once I replace the user-definable colors with selectable color themes, of course retaining the custom option.

That way a color theme will be actually two sets of colors, a dark and a light variant, so the application can pick the right one depending on the user's system theme.

**Comment by carrotIndustries** [comment]
included as the default dark theme since 9c6945ed9a9c85fb6135baa6344f0662a8b7a1c4

**Comment by jue89** [comment]
Nice! I feel honored ☺️
